homeassistant/components/kodi/browse_media.py

```diff
@@ -32,16 +32,18 @@
     MEDIA_TYPE_PLAYLIST,
     MEDIA_TYPE_TVSHOW,
     MEDIA_TYPE_SEASON,
+    "library_music",
 ]
 
 CONTENT_TYPE_MEDIA_CLASS = {
-    "library_music": MEDIA_CLASS_MUSIC,
+    "library_music": MEDIA_CLASS_DIRECTORY,
```

Comment:
What does this library contain? Is it other types than music? We have defined `MEDIAL_CLASS_DIRECTORY` as an item with children where we can't define the media type. If we can define it as music, we should not use `MEDIA_CLASS_DIRECTORY`.

Reply:
Addressed in #39840